base/index.js

```diff
@@ -28,21 +28,9 @@ module.exports = (opts) => {
     {p: 'tables', n: 'migrations'},
     {p: 'tests/middleware', n: 'jwt'},
     {p: 'tests/routes', n: 'schema'},
-  ].map(thing => _.extend({p: '', d: thing.p || '', t: thing.n, e: 'js'}, thing));
-
-  things.forEach(thing => {
-    const templatePath = path.join(__dirname, thing.p, thing.n+'.mustache');
-    const template = fs.readFileSync(templatePath).toString();
-
-    const dir = path.join(process.cwd(), thing.d);
-    const target = path.join(dir, thing.t+'.'+thing.e);
-
-    if (fs.existsSync(target)) return console.error(target, 'already exists. Not overwriting it');
+  ].map(file => _.extend({p: '', d: file.p || '', t: file.n, e: 'js'}, file));
```

Comment:
I think we should move this `map` and `extend` into the `scaffold` lib, as very similar in all three places.